### PR TITLE
155139484 upload

### PR DESCRIPTION
**Pull request by CodingCarlos**
Merged dev before the first commit.

### CODE REVIEW

**Comment by CodingCarlos** [comment]
Don't merge yet. Tests are failing because the file drag library is not in a CDN, and also are not being auto-copied to /dist. I'm doing some gulp magic, and it will be done asap.

**Comment by CodingCarlos** [comment]
Now, it shall be working. I've got to update docs to add this thing of joining dependencies without public CDNs. Basically, just add the module's file path to gulp dependencies array (`sourceDeps`).

File: package.json
Comment (DevStarlight) [inline package.json:46]: can you explain for what are you using this library?

File: pages/upload/upload.view.scss
Comment (DevStarlight) [inline pages/upload/upload.view.scss]: use @extend to provide full compatibility with browsers

File: pages/upload/upload.view.scss
Comment (DevStarlight) [inline pages/upload/upload.view.scss]: same `@extend display-flex` to provide compatibility

File: pages/upload/upload.view.scss
Comment (DevStarlight) [inline pages/upload/upload.view.scss]: same @extend display-flex to provide compatibility

File: pages/upload/upload.view.scss
Comment (DevStarlight) [inline pages/upload/upload.view.scss:65]: we already have a `@include border-radius(4px)` for this

**Review by DevStarlight — CHANGES_REQUESTED** [review]
After doing some QA to this story checking that it works, It lacks some files

**Comment by DevStarlight** [comment]
I don't find test uploaded @CodingCarlos, by the way, looks like there is a huge work executed correctly! Well done!

File: package.json
Comment (CodingCarlos) [reply package.json:46]: [It was already in dev]( https://github.com/Videona/platform-frontend/blob/dev/package.json#L49), added in 1096efe ([see line](https://github.com/Videona/platform-frontend/commit/1096efe61977cc6dffeedc2aa23c164c98b3d708#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R47)). 

Anyway, it's being used for gulp to [pipe angular app configs in a file](https://github.com/Videona/platform-frontend/commit/1096efe61977cc6dffeedc2aa23c164c98b3d708#diff-b9e12334e9eafd8341a6107dd98510c9R34) ;)

**Comment by CodingCarlos** [comment]
Your're right. I'm changing this things :dog:

**Comment by CodingCarlos** [comment]
Well, it's done! Also, i've added right product types, and moved names to language files.

**Comment by DevStarlight** [comment]
It seems all is fine =) Good Job @CodingCarlos!!

**Review by DevStarlight — APPROVED** [review]

**Review by javierLiarte — COMMENTED** [review]
I've couldn't review yet, but trying to deploy locally, I get this error on buiñd frontend:

Error in plugin 'sass'
Message:
    sass/helpers/_placeholders.scss
Error: You may not @extend an outer selector from within @media.
       You may only @extend selectors within the same directive.
       From "@extend %flex-direction-row" on line 28 of pages/upload/upload.view.scss
        on line 95 of sass/helpers/_placeholders.scss
>> %flex-direction-row{

File: pages/upload/upload.view.scss
Comment (DevStarlight) [inline pages/upload/upload.view.scss:22]: It's not possible to include @extend directives in an @media and... due to we're @include a @media, this is not allowed

**Comment by DevStarlight** [comment]
@javierLiarte I did a second review focusing in the .scss and I found the problem at including @extend directives in @media as I commented before in a previous comment

**Comment by CodingCarlos** [comment]
Ready to QA test now, thanks!

**Comment by javierLiarte** [comment]
Now frontend image builds correctly, but when I sart page on browser I get this error:

`angular.js:88 Uncaught Error: [$injector:modulerr] http://errors.angularjs.org/1.6.5/$injector/modulerr?p0=app&p1=Error%3A%20%5B%24injector%3Amodulerr%5D%20http%3A%2F%2Ferrors.angularjs.org%2F1.6.5%2F%24injector%2Fmodulerr%3Fp0%3DngFileUpload%26p1%3DError%253A%2520%255B%2524injector%253Anomod%255D%2520http%253A%252F%252Ferrors.angularjs.org%252F1.6.5%252F%2524injector%252Fnomod%253Fp0%253DngFileUpload%250A%2520%2520%2520%2520at%2520https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A7%253A76%250A%2520%2520%2520%2520at%2520https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A26%253A408%250A%2520%2520%2520%2520at%2520b%2520(https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A25%253A439)%250A%2520%2520%2520%2520at%2520https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A26%253A182%250A%2520%2520%2520%2520at%2520https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A42%253A290%250A%2520%2520%2520%2520at%2520p%2520(https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A8%253A7)%250A%2520%2520%2520%2520at%2520g%2520(https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A42%253A138)%250A%2520%2520%2520%2520at%2520https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A42%253A322%250A%2520%2520%2520%2520at%2520p%2520(https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A8%253A7)%250A%2520%2520%2520%2520at%2520g%2520(https%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.6.5%252Fangular.min.js%253A42%253A138)%0A%20%20%20%20at%20https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A7%3A76%0A%20%20%20%20at%20https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A43%3A70%0A%20%20%20%20at%20p%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A8%3A7)%0A%20%20%20%20at%20g%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A42%3A138)%0A%20%20%20%20at%20https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A42%3A322%0A%20%20%20%20at%20p%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A8%3A7)%0A%20%20%20%20at%20g%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A42%3A138)%0A%20%20%20%20at%20gb%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A46%3A251)%0A%20%20%20%20at%20c%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A22%3A19)%0A%20%20%20%20at%20Uc%20(https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.6.5%2Fangular.min.js%3A22%3A332)
    at angular.js:88
    at angular.js:4957
    at p (angular.js:410)
    at g (angular.js:4917)
    at gb (angular.js:4839)
    at c (angular.js:1949)
    at Uc (angular.js:1970)
    at xe (angular.js:1855)
    at angular.js:33826
    at HTMLDocument.b (angular.js:3468)
(anonymous) @ angular.js:88
(anonymous) @ angular.js:4957
p @ angular.js:410
g @ angular.js:4917
gb @ angular.js:4839
c @ angular.js:1949
Uc @ angular.js:1970
xe @ angular.js:1855
(anonymous) @ angular.js:33826
b @ angular.js:3468`

**Comment by CodingCarlos** [comment]
Let's see if now it's working...

**Comment by javierLiarte** [comment]
Updated product type strings to those defined in https://www.pivotaltracker.com/story/show/155550527

**Pull request merged**
